### DISCUSSION TITLE
How to register new user with code or API?

### DISCUSSION THREAD

**Discussion by steve2bi** [simpleidserver/SimpleIdServer#492] [General]
without idServer website, how to register new user? Thanks!

**Comment by simpleidserver** [comment]
Hello,

Unfortunately there is no api to manage users. 
We add a new ticket #493 to support this feature. It will be included in the next release. 

It is still possible to add users in code, by using the class Userrepository  : https://github.com/simpleidserver/SimpleIdServer/blob/9746f32fcfaf327f9c6e8d9f9f1a2cb129a4160d/src/IdServer/SimpleIdServer.IdServer.Store/UserRepository.cs

Kr,

Sid

**Comment by simpleidserver** [comment]
The API is developed and is available in the branch `release/v3.0.4`.
